index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio page</title>
+</head>
+<body>
+    <section id="welcome-section">
+        <h1>Hi there , I am Joey</h1>
+        
+    </section>
+    <section id="projects">
+        <p class="project-tile" id="project-tile"></p>
```

Comment:
- [ ] Impressive work for phrasing your comment politely. It would be better if you provided more context to let the student know why adding a paragram text is necessary. This will help the student to learn from their mistakes and not repeat them again in their future projects.😊